src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  column-gap: 4px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
```

Comment:
It is a good practice to set a default background image for the `.stars__star` class. This will ensure that there is always a star displayed, even if no modifier class is applied. You can set the default background image to the inactive star image.